modules/puzzle/src/main/Attempt.scala

```diff
@@ -34,40 +30,34 @@ object Attempt {
     val date = "d"
     val win = "w"
     val time = "t"
```

Comment:
Not really imo. Because the time taken doesn't have any impact on rating, it also has no meaning: players just sit on puzzle pages and/or multitask it. I don't see how such weak data can be used for meaningful analytics.